tenacity/tests/test_tenacity.py

```diff
@@ -629,6 +629,17 @@ def test_retry_if_exception_message_negative_too_many_inputs(self):
             tenacity.retry_if_exception_message(
                 message="negative", match="negative")
 
+    def test_retry_kwargs_passed_properly(self):
+
+        def _kw_fun():
+            raise tenacity.TryAgain
+
+        g = retry(_kw_fun, stop=tenacity.stop_after_attempt(5), reraise=True)
```

Comment:
Hm, I'm not sure about that. It's not supposed to be used like that. It's supposedly a decorator.

You're supposed to do this here:

```python
g = retry(stop=tenacity.stop_after_attempt(5), reraise=True)(_kw_fun)
```

Reply:
The syntax seems to support both but I swapped the test to be consistent 4d5b3fa